src/v/kafka/server/usage_aggregator.h

```diff
@@ -43,8 +43,9 @@ std::chrono::time_point<clock_type, duration> round_to_interval(
     vlog(
       klog.error,
       "usage has detected a timestamp '{}' that exceeds the preconfigured "
-      "threshold of 2min meaning a clock has fired later or earlier then "
+      "threshold of {}s meaning a clock has fired later or earlier then "
```

Comment:
above the `2min` constants didn't change, but here it changes. was the comment wrong?

Reply:
the constant is 2minutes is the interval is greater then 2 minutes, otherwise it was the interval. This is only true in ducktape tests where the interval was set to 3 seconds. This is as wide as the entire window itself, and it was essentially covering up the clock skew that existed. Only when skew was greater then a window would a test failures be observed